test/iron-iconset-svg.html

```diff
@@ -134,6 +136,38 @@
           iconset.applyIcon(div, 'circle');
           expect(div.firstElementChild.getAttribute('viewBox')).to.be.equal('0 0 20 20');
         });
+      });
+
+      suite('Adding / removal from iron-icon', function () {
+        var iconset;
+        var div;
+        var ironIcon;
+
+        setup(function () {
+          var elements = fixture('StandardIconsetSvg');
+          iconset = elements[0];
+          div = elements[1];
+          ironIcon = elements[2];
+        });
+
+        test('be able to remove an iconset from a standard DOM element', function () {
+          iconset.applyIcon(div, 'circle');
+          Polymer.dom.flush()
```

Comment:
Missing `;` here and on your other calls to `flush` - certainly not worth blocking the PR on this though.